scripts/packages_exporter.py

```diff
@@ -365,14 +368,17 @@ async def generate_rss(self, platform, modern_cache):
         dist_version = platform.split('-')[-1]
 
         errata_data = modern_cache['data']
+        sorted_errata_data = sorted(
+            erratum_data, key=lambda k: k['updated_date'], reverse=True
```

Comment:
s/erratum_data/errata_data ?

Reply:
Yess, it slipped out during squash & resolving conflicts.